src/SlimSession/Helper.php

```diff
@@ -13,6 +13,16 @@
 class Helper implements \ArrayAccess, \Countable, \IteratorAggregate
 {
     /**
+     * Get all session variables.
+     *
+     * @return array
+     */
+    public function getAll()
+    {
+        return $_SESSION;
```

Comment:
It'd be nice if the return value is inmutable. :smile: So `$_SESSION` values can't be overriden by accident.